vhdbuilder/packer/install-dependencies.sh

```diff
@@ -416,13 +416,18 @@ K8S_VERSIONS="
 1.15.7-hotfix.20200326
 1.15.10-hotfix.20200408.1
 1.15.11-hotfix.20200817.1
+1.15.11-hotfix.20200824.1
 1.15.12-hotfix.20200817.1
+1.15.12-hotfix.20200824.1
 1.16.9-hotfix.20200529.1
 1.16.10-hotfix.20200817.1
+1.16.10-hotfix.20200824.1
 1.16.13-hotfix.20200817.1
+1.16.13-hotfix.20200824.1
 1.17.3-hotfix.20200601.1
 1.17.7-hotfix.20200817.1
 1.17.9-hotfix.20200817.1
+1.17.9-hotfix.20200824.1
```

Comment:
for kubelet and kubectl we just need the latest version on the node. It's different than hyperkube that is decided by ccp charts

Reply:
got it, updated